### ISSUE TITLE
Chrome Lightouse Audit - Performance, Accessibility & SEO Issues

### ISSUE THREAD

**Issue by ychoy** [codeforsanjose/free-food-lookup#2]
Recommendation as a baseline to check for performance, accessibility, SEO, and best practices. Checking for these issues should be continuous. 

Current score according to Chrome Lighthouse Audit (as of May 1, 2020)
- 98 for Performance
- 100 for Accessibility 
- 100 for SEO
